pb-jelly-gen/codegen/codegen.py

```diff
@@ -1562,7 +1562,7 @@ def get_cargo_toml_file(self, derive_serde: bool) -> Iterator[Tuple[Text, Text]]
             versions = {
                 u"lazy_static": u' version = "1.4.0" ',
                 u"pb-jelly": u' version = "0.0.5" ',
-                u"serde": u' version = "1.0.114" ',
+                u"serde": u' version = "1.0" ',
                 u"bytes": u' version = "0.5.6" '
```

Comment:
I assume we'll want to do this too at some point? It's going to be confusing the be inconsistent.